.github/workflows/build.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       labels: bottlerocket_ubuntu-latest_16-core
     steps:
       - uses: actions/checkout@v3
-      - run: rustup default 1.61.0 && rustup component add rustfmt && rustup component add clippy
+      - run: rustup default 1.62.1 && rustup component add rustfmt && rustup component add clippy
```

Comment:
Since the pinned version is getting called in two places would it make sense to just make it a variable?

Also we probably want to pin to the same rust version as the `bottlerocket-sdk` _(currently 1.66.1)_.